.github/workflows/ci.yml

```diff
@@ -32,13 +32,23 @@ jobs:
         with:
           distribution: corretto
           java-version: 11
+          cache: sbt
 
       - name: build + test
-        run: |
-          set -e
-          # Ensure we don't overwrite existing (Teamcity) builds.
-          LAST_TEAMCITY_BUILD=2360
-          export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
+        env:
+          # The GitHub runner has 7GB of RAM, lets give 4GB to Java.
+          # See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.
+          JAVA_OPTS: -Xmx4g
```

Comment:
Previously, we have been using the [sbt file in the repository](https://github.com/guardian/riff-raff/blob/4b06be7fc4566e81d951adf87837a30282725513/sbt) which set the JVM flags `-Xmx768M -XX:+UseCompressedOops -XX:MaxPermSize=384m`.

For simplicity, we just set `Xmx` (maximum heap size) now as `MaxPermSize` is deprecated and `UseCompressedOops` is defaulted to on already.

The value of `Xmx` is different (768M vs 4g) to take advantage of the GitHub runner resources; we have 7GB available, so might as well use (most of) it!